src/webview/js/js.js

```diff
@@ -679,6 +680,14 @@ documentBody.addEventListener('click', (e: MouseEvent) => {
     return;
   }
 
+  if (target.closest('.spoiler-header')) {
+    const spoiler = target.closest('.spoiler-block');
+    if (spoiler instanceof HTMLElement) {
+      toggleSpoiler(spoiler);
+    }
+    return;
```

Comment:
> we can drop the talk of `.spoiler-header` in this chunk of code entirely.

This might have been in response to an earlier revision and I'm not following along so well, but I notice that there's still talk of `.spoiler-header` in the current revision, and I mention a reason for it at https://github.com/zulip/zulip-mobile/pull/4169#discussion_r518970426. Is there still something pending from your comment here?

Reply:
Yeah, the remaining discussion related to this is at that thread.

(The thing this thread was specifically about is fixed in the current revision: the `return` is inside the inner `if`.)